### PR TITLE
fix cant send empty message by pressing enter

### PR DESCRIPTION
**Pull request by yermolenko-d**

### CODE REVIEW

File: frontend/src/app/modules/group/group-chat/group-chat.component.ts [hunk 1]
```diff
@@ -73,6 +73,9 @@ export class GroupChatComponent extends BaseComponent implements OnInit, AfterVi
   }
 
   sendMessage(){
+    if (this.textOfMessage === ''){
```
Comment (VoBilyk) [inline frontend/src/app/modules/group/group-chat/group-chat.component.ts:76]: Could you disable the button instead of this?

**Review by VoBilyk — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/group/group-chat/group-chat.component.ts (hunk 1 above)
Comment (yermolenko-d) [reply frontend/src/app/modules/group/group-chat/group-chat.component.ts:76]: it's disabled, but there is actually an opportunity to send messages by pressing "enter"

Comment (yermolenko-d) [reply frontend/src/app/modules/group/group-chat/group-chat.component.ts:76]: so this case is only for this option

Comment (VoBilyk) [reply frontend/src/app/modules/group/group-chat/group-chat.component.ts:76]: OK

**Pull request merged**